addons/website/models/ir_ui_view.py

```diff
@@ -134,12 +134,10 @@ def _load_records_create(self, values):
                     ('website_id', '!=', None),
                 ])
                 for specific_parent_view in specific_parent_views:
-                    record.copy({
-                        # Set key to avoid copy() to generate an unique key as
-                        # we want the specific view to have the same key
-                        'key': record.key,
+                    website_id = specific_parent_view.website_id.id
+                    record.with_context(website_id=website_id).write({
                         'inherit_id': specific_parent_view.id,
-                        'website_id': specific_parent_view.website_id.id,
+                        'website_id': website_id,
```

Comment:
You do not need the 'website_id' key right ?

Reply:
Indeed, that line was no more useful as it would be added during COW from context